broker/makeMqttServer.js

```diff
@@ -5,22 +5,20 @@ var mosca = require('mosca');
 var debug = require('../tools/debug');
 
 var pubsubsettings = {
-  type: 'redis',
-  redis: require('redis'),
-  db: 12,
-  host: process.env.REDIS_PORT_6379_TCP_ADDR,
-  port: process.env.REDIS_PORT_6379_TCP_PORT,
-  return_buffers: true
+    type: 'redis',
+    redis: require('redis'),
+    db: 12,
+    host: process.env.REDIS_PORT_6379_TCP_ADDR || 'localhost',
+    return_buffers: true
```

Comment:
no port? we can simplify the name of the variable if there is no more docker. Or should we keep it for the docker test?